plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskProjectWizard.java

```diff
@@ -191,10 +191,14 @@ private void doFinish(String containerName, String fileName, String taskName, St
                     HumantaskEditorConstants.ERROR_CREATING_INITIAL_FILE_MESSAGE, editorStatus);
         } finally {
             try {
-                stream.close();
-                wsdlStream.close();
-                htconfigStream.close();
-                orgSchemaStream.close();
+                if (stream != null)
```

Comment:
Thanks. Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/30/commits/acbad877337a7e63dd99f935d9d2c3329ab6c1de